packages/@aws-cdk/cdk-deploy/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "@aws-cdk/cdk-deploy",
+  "description": "Command-line utility for deploying CDK applications",
+  "version": "0.28.0",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "bin": {
+    "cdk-deploy": "bin/cdk-deploy"
+  },
+  "scripts": {
+    "build": "cdk-build",
+    "watch": "cdk-watch",
+    "lint": "cdk-lint",
+    "pkglint": "pkglint -f",
+    "test": "cdk-test",
+    "package": "cdk-package"
+  },
+  "nyc": {
+    "statements": 8,
+    "lines": 8,
+    "branches": 3
+  },
+  "author": {
+    "name": "Amazon Web Services",
+    "url": "https://aws.amazon.com",
+    "organization": true
+  },
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@types/archiver": "^2.1.2",
+    "@types/fs-extra": "^5.0.4",
+    "@types/minimatch": "^3.0.3",
+    "@types/mockery": "^1.4.29",
+    "@types/request": "^2.47.1",
+    "@types/semver": "^5.5.0",
+    "@types/sinon": "^7.0.10",
+    "@types/table": "^4.0.5",
+    "@types/uuid": "^3.4.3",
+    "@types/yaml": "^1.0.0",
+    "@types/yargs": "^8.0.3",
+    "cdk-build-tools": "^0.28.0",
+    "mockery": "^2.1.0",
+    "pkglint": "^0.28.0",
+    "sinon": "^7.2.7"
+  },
+  "dependencies": {
```

Comment:
Might be a bit tricky to do at the moment, but one of the goal is to dramatically reduce dependencies for this module given it's sensitivity. For example, I don't think this module needs any fancy CLI/colors/yargs stuff. It should be a low-level utility that is executed by other tools (maybe JSON STDIN, like lambda builders).

Okay if this is too much for the current project, but if there are low-hanging fruit, it would be nice to start cleaning up.